docs/howto/openstack/neutron/create-security-groups.md

```diff
@@ -0,0 +1,59 @@
+# How to Create Security Groups
+
+By definition, security groups are "[...] _sets of IP filter rules that are applied to all project instances, which define networking access to the instance. Group rules are project specific; project members can edit the default rules for their group and add new rule sets_". [[1](https://docs.openstack.org/nova/latest/admin/security-groups.html)]
+
+To create a security group click on "_Security Groups_" in the left-hand navigation menu:
```

Comment:
Leave the quotation marks out. We have adopted the convention that when we refer to labels and buttons from the GUI, we just put them in italics and don't quote them.